protons/driver.py

```diff
@@ -754,6 +755,110 @@ def _cache_force(self, titration_group_index, titration_state_index):
 
         self.titrationGroups[titration_group_index]['titration_states'][titration_state_index]['forces'] = f_params
 
+    def _ncmc_ghmc(self, context, titration_group_indices, initial_titration_states, final_titration_states):
+        """
```

Comment:
Could you add to the docstring what the expected behavior is when you call this function?

e.g. "Attempt a single NCMC protonation state change."